services/brig/test/integration/API/User/Auth.hs

```diff
@@ -90,30 +95,18 @@ testEmailLogin brig = do
     let Just email = userEmail u
     now <- liftIO getCurrentTime
 
-    -- Login with email
+    -- Login with email and do some checks
     _rs <- login brig (defEmailLogin email) PersistentCookie
         <!! const 200 === statusCode
-
-    -- Check basic cookie attributes
-    let ck = decodeCookie _rs
-    liftIO $ do
-        assertBool "type" (cookie_persistent ck)
-        assertBool "http-only" (cookie_http_only ck)
-        assertBool "expiry" (cookie_expiry_time ck > cookie_creation_time ck)
-        assertBool "domain" (cookie_domain ck /= "")
-        assertBool "path" (cookie_path ck /= "")
-
-    -- Check basic access token attributes
-    let tk = decodeToken  _rs
     liftIO $ do
-        assertEqual "user" (userId u) (ZAuth.accessTokenOf tk)
-        assertBool "expiry" (ZAuth.tokenExpiresUTC tk > now)
+        assertSanePersistentCookie (decodeCookie _rs)
```

Comment:
If we are using `_rs`, it should be renamed to `rs`, no?

Reply:
(same below)

Reply:
It's used in about 30 places in code. I think in those cases it's supposed to mean "this is temporary" and not "this is unused", but I agree that it's slightly weird.